test/forward.test.cpp

```diff
@@ -738,6 +750,24 @@ TEST_CASE("Eigen::VectorXdual tests", "[dual]")
         REQUIRE( g[2] == approx(x[2]) );
     }
 
+    SECTION("testing gradient derivatives with Erf")
+    {
+        auto f = [](const VectorXdual& x) -> dual
+        {
+            return erf(x.array()).sum();
```

Comment:
So, there is no `x.array().erf()`?

Reply:
I checked, and the following  `x.array().erf()` seems valid but causes the same compilation error with Eigen 3.7.